snoop/utils.py

```diff
@@ -10,6 +10,12 @@ def chunks(file, blocksize=65536):
 def pdftotext(input):
     return subprocess.check_output(['pdftotext', '-', '-'], stdin=input)
 
+def build_raw_query(where, table, field='id'):
```

Comment:
I think we can leave the `field` argument for later, it's not used anywhere, and it makes sense to reverse the other two, so I'd define the function like so:
```python
def user_select_sql(table, where):
    """Build a raw SQL query with a user-defined `where` clause."""
```